src/tests/ffi.cpp

```diff
@@ -2106,7 +2488,7 @@ TEST_F(rnp_tests, test_ffi_key_generate_protection)
 
     /* Generate key and subkey without protection */
     rnp_op_generate_t keygen = NULL;
-    assert_rnp_success(rnp_op_generate_create(&keygen, ffi, "RSA"));
+    assert_rnp_success(rnp_op_generate_create(&keygen, ffi, PGP_ALGNAME_RSA));
```

Comment:
I would suggest to leave string constants instead define usage so we'll have more coverage on misc case usage.